lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1199,15 +1203,16 @@ mkApiWallet
     -> WalletMetadata
     -> SyncProgress
     -> Set Tx
+    -> (Quantity "lovelace" Word64)
     -> ApiWallet
-mkApiWallet wid wallet meta progress pending = ApiWallet
+mkApiWallet wid wallet meta progress pending rewards = ApiWallet
```

Comment:
**[question]**

Why a plural form here :grimacing:  ?

Reply:
Without `balance` as suffix, I thought `rewards` sounded better than `reward`. (@jonathanknowles ?)

`rewardBal[ance]` might be a safe bet though.

Reply:
Either "reward" or "rewards" are fine, I think!

However, perhaps "rewards" has the unwanted connotation of implying a list or some other container.

`rewardBalance` is also a good compromise.